src/shipit_static_analysis/tests/test_reporter_mail.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import responses
+import pytest
+import json
+
+
+@responses.activate
+def test_conf(mock_config):
+    '''
+    Test mail reporter configuration
+    '''
+    from shipit_static_analysis.report.mail import MailReporter
+
+    # Missing emails conf
+    with pytest.raises(AssertionError):
+        MailReporter({}, 'test_tc', 'token_tc')
+
+    # Missing emails
+    conf = {
+        'emails': [],
+    }
+    with pytest.raises(AssertionError):
+        MailReporter(conf, 'test_tc', 'token_tc')
+
+    # Valid emails
+    conf = {
+        'emails': [
+            'test@mozilla.com',
+        ],
+    }
+    r = MailReporter(conf, 'test_tc', 'token_tc')
+    assert r.emails == ['test@mozilla.com', ]
+
+    conf = {
+        'emails': [
+            'test@mozilla.com',
+            'test2@mozilla.com',
+            'test3@mozilla.com',
```

Comment:
Hopefully these won't spam actual people at Mozilla. 🙂

Reply:
The unit tests do not have access to Taskcluster (dummy credentials), so no mail is *really* sent.